### ISSUE TITLE
loss of the validation set always nan

### ISSUE THREAD

**Issue by bo-jpg** [Coldwave96/MaliciousUrls-DL#1]
Thanks for your work!

Why is the loss of the validation set always nan and the accuracy remains unchanged during training? (The loss and accuracy on the training set change normally)

Looking forward to your reply!
